CMakeLists.txt

```diff
@@ -13,7 +13,9 @@ set(CUKE_ENABLE_EXAMPLES OFF CACHE BOOL "Enable the examples")
 set(GMOCK_SRC_DIR "" CACHE STRING "Google Mock framework sources path (otherwise downloaded)")
 set(GMOCK_VER "1.7.0" CACHE STRING "Google Mock framework version to be used")
 option(VALGRIND_TESTS "Run tests within Valgrind" OFF)
+set(CUKE_DISABLE_QT OFF CACHE BOOL "Disable using Qt framework")
 
+set(ignoreMe "${QT_QMAKE_EXECUTABLE}") #supress warning
```

Comment:
I don't think it's a good idea to silence that warning. Because when it gives that warning the parameter you passed on the command line actually isn't used. If that's okay under some circumstances then I would just allow the warning to be displayed.